test/frame_test.dart

```diff
@@ -223,12 +223,8 @@ void main() {
           (text) => Frame.parseV8(text), '    at Foo (dart:async/future.dart)');
       expectIsUnparsed((text) => Frame.parseV8(text),
           '    at (dart:async/future.dart:10:15)');
-      expectIsUnparsed(
```

Comment:
can this test case remain? I'd expect we'd continue to leave this unparsed.

Reply:
returned